src/Views/FirmwareView.vala

```diff
@@ -354,8 +354,8 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
 
     private void show_reboot_dialog () {
         var message_dialog = new Granite.MessageDialog.with_image_from_icon_name (
-            _("An update requires the system to restart to complete"),
-            _("This will close all open applications and restart this device."),
+            _("Installing new firmware requires a system reboot to complete"),
```

Comment:
As the wingpanel indicator uses the words "Shutdown ... Restart", I do not see the advantage of using the technical word "reboot" instead.

Reply:
Says "Reboot" in LXQt. Reboot is always start something up again in a booting fashion, whereas "restart" could be something just unloading and starting again.
Guess I am reverting this.

Reply:
I guess it's a good thing we use Pantheon and not LXQt :) We should definitely be internally consistent here and use "Restart"